pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -156,13 +158,34 @@ func (a *ApplicationBackupController) performRuleRecovery() error {
 	return lastError
 }
 
-func (a *ApplicationBackupController) setDefaults(backup *stork_api.ApplicationBackup) {
+func (a *ApplicationBackupController) setDefaults(backup *stork_api.ApplicationBackup) bool {
+	updated := false
 	if backup.Spec.ReclaimPolicy == "" {
 		backup.Spec.ReclaimPolicy = stork_api.ApplicationBackupReclaimPolicyDelete
+		updated = true
 	}
 	if backup.Status.TriggerTimestamp.IsZero() {
 		backup.Status.TriggerTimestamp = backup.CreationTimestamp
+		updated = true
+	}
+	return updated
+}
+
+func (a *ApplicationBackupController) updateWithAllNamespaces(backup *stork_api.ApplicationBackup) error {
+	namespaces, err := core.Instance().ListNamespaces(nil)
+	if err != nil {
+		return fmt.Errorf("error updating with all namespaces for wildcard: %v", err)
```

Comment:
nit:  as this is listing, should `updating` be changed to `listing`

Reply:
The function is trying to update the backup spec with all the namespaces